certified-connectors/UiPath Orchestrator/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3420 @@
+                      ],
+                      "type": "string",
+                      "x-ms-enum":
+                      {
+                        "name": "StartProcessDtoRuntimeType",
+                        "modelAsString": false
+                      }
+                    },
+                    "InputArguments":
+                    {
+                      "description": "Input parameters in JSON format to be passed to job execution.",
+                      "maxLength": 10000,
+                      "type": "string"
+                    },
+                    "Reference":
+                    {
+                      "description": "Optional user-specified reference for jobs",
+                      "maxLength": 128,
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            }
+          },
+          {
+            "in": "header",
+            "name": "X-UIPATH-OrganizationUnitId",
+            "description": "Folder/OrganizationUnit Id",
+            "type": "integer",
+            "format": "int64",
+            "required": true,
+            "x-ms-summary": "Folder",
+            "x-ms-dynamic-values":
+            {
+              "operationId": "Folders_Get",
+              "value-path": "Id",
+              "value-collection": "value",
+              "value-title": "FullyQualifiedName"
+            }
+          }
+        ],
+        "produces":
+        [
+          "application/json"
+        ],
+        "responses":
+        {
+          "201":
+          {
+            "description": "Success",
+            "schema":
+            {
+              "$ref": "#/definitions/ODataValueOfIEnumerableOfJobDto"
+            }
+          }
+        },
+        "summary": "Start Job",
+        "tags":
+        [
+          "Jobs"
+        ]
+      }
+    },
+    "/odata/Processes":
+    {
+      "get":
+      {
+        "tags":
+        [
+          "Processes"
+        ],
+        "summary": "Get Processes",
+        "description": "Gets a list of processes.",
+        "operationId": "Processes_Get",
+        "x-ms-visibility": "internal",
+        "produces":
+        [
+          "application/json"
+        ],
+        "parameters":
+        [
+          {
+            "in": "query",
+            "name": "searchTerm",
+            "description": "Used to narrow result items to those matching the requested search term.",
+            "type": "string",
+            "default": "",
+            "x-ms-summary": "Search Term",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "in": "query",
+            "name": "feedId",
+            "description": "Indicates the feed id.",
+            "type": "string",
+            "format": "uuid",
+            "x-ms-summary": "Search Term",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "in": "query",
+            "name": "$filter",
```

Comment:
Since you are using these parameters ($filter, $orderby, $top) it makes sense to use `parameters` like this connector: https://github.com/microsoft/PowerPlatformConnectors/blob/5a304e21c47b594bc18500252ae9f63634956d85/certified-connectors/Azure%20Communication%20Services%20Chat/apiDefinition.swagger.json#L673

Reply:
Thanks, moved to parameters.